apps/user/tests/test_schemas.py

```diff
@@ -587,7 +587,8 @@ def _query_check(filters, **kwargs):
             ('exclude-project-2', dict(membersExcludeProject=project2.pk), 2, [user1, user3]),
             ('exclude-af-1', dict(membersExcludeFramework=af1.pk), 1, [user3]),
             ('exclude-af-2', dict(membersExcludeFramework=af2.pk), 3, [user, user1, user3]),
-            ('search', dict(search='Guy'), 1, [user]),
+            ('search-1', dict(search='Normal'), 1, [user]),
+            ('search-2', dict(search='Guy'), 1, [user]),
```

Comment:
I think we are still missing test cases related to this issue. Let's add more cases like
Search text:
- Firstname Lastname
- FIrstname
- Lastname
- <spaces>Firstname
- Firstname<spaces>
- Firstname Lastname<spaces>
- Email
- Email(Partial text only)

Reply:
@thenav56 ,
Is there a difference between the 2nd,4th & 5th Firstnames ?
and also 1st & 6th Firstname Lastname ?

Reply:
Hi @RobertKoikai, yes there is but I didn't check the markdown preview. Here is the raw text
```
Search text:
- Firstname Lastname
- FIrstname
- Lastname
- <spaces>Firstname
- Firstname<spaces>
- Firstname Lastname<spaces>
- Email
- Email(Partial text only)
```